docs/help.rst

```diff
@@ -0,0 +1,97 @@
+.. include:: links.inc
+
+.. _help:
+
+Getting Help
+============
+
+    | Help! I need somebody
+    | Help! not just anybody
+    | Help! you know I need someone
+    | Help!
+
+    -- The Beatles, Help!
+
+This fragment of the song originally appeared in the 1965 film Help! and was written by Lennon and McCartney, just a
+few years after the development of Mark 1, the first commercial computer. Maybe just a random coincidence, or maybe
+Lennon was just trying to write an email to McCartney.
+
+While the song might have been written with another meaning in mind, it is a good way to describe the situation of many
+researchers who are presented with a new toolbox. Indeed, the situation of many researchers is that the projects they
+are working on are becoming more and more complex in terms of methods and data. Thus, we *open up the doors* to new 
+possibilites:
+
+    | When I was younger, so much younger than today
+    | I never needed anybody's help in any way
+    | But now these days are gone, I'm not so self assured
+    | Now I find I've changed my mind and opened up the doors
+
+    -- The Beatles, Help!
+
+The setback with modern reseearch is that nowaday's methods are often more complex and require more computing, which
+means that we need to learn concepts from computer sciences, mathematics, statistics, etc. This is a good thing, but it
+also means that we need to learn new tools and new ways of thinking, which can be a bit overwhelming, to the point that
+we start relying more and more on other researchers. In the end, we might feel like we loose our independence:
+
+    | And now my life has changed in oh so many ways
+    | My independence seems to vanish in the haze
+    | But every now and then I feel so insecure
+    | I know that I just need you like I've never done before
+
+    -- The Beatles, Help!
+
+We can continue with the song, but we think you get the point. The point is that we will need help, and we will need
+to ask for it. We will need to ask for help from our colleagues, from our supervisors, from our friends.
+
+We are a small team of researchers and developers, and we are not experts in *everything at once*. Each one of us has
+a specific expertise, and we are trying to use this expertise to create Junifer. When we thought of Junifer, we thought
```

Comment:
I propose: "When we conceived Junifer, we thought of neuroimaging researchers' problems, and tried to come up with the best way to help them by building a tool that is easy to understand, learn and use. And of course, we are here to help you with your research."